src/azure-cli/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -238,7 +238,13 @@
 helps['keyvault create'] = """
 type: command
 short-summary: Create a Vault or HSM.
-long-summary: If `--enable-rbac-authorization` is not specified, then default permissions are created for the current user or service principal unless the `--no-self-perms` flag is specified.
+long-summary: >-
+    Default permissions are created for the current user or service principal unless the `--no-self-perms`
+    or `--enable-rbac-authorization` flag is specified.
```

Comment:
```suggestion
    RBAC authorization is enabled by default. If `--enable-rbac-authorization` is manually specified to `false`, then default permissions are created for the current user or service principal unless the `--no-self-perms` flag is specified.
```

Reply:
> RBAC authorization is enabled by default.

I saw this change was made in #28886, but this help message is not updated (https://github.com/Azure/azure-cli/pull/28886#issuecomment-2327889442).

```suggestion
    RBAC authorization is enabled by default. If `--enable-rbac-authorization` is manually specified to `false` and 
    `--no-self-perms` flag is not specified, default permissions are created for the current user or service principal.
```

Reply:
Yes, my bad. I just realized when reviewing your PR. You are so careful❤️